junit-platform-commons/src/main/java/org/junit/platform/commons/support/AnnotationSupport.java

```diff
@@ -133,6 +133,23 @@ public static boolean isAnnotated(AnnotatedElement element, Class<? extends Anno
 		return AnnotationUtils.findRepeatableAnnotations(element, annotationType);
 	}
 
+	/**
+	 * Find all {@code public} {@linkplain Field fields} of the supplied class
+	 * or interface that are annotated or <em>meta-annotated</em> with the specified {@code annotationType}.
+	 *
+	 * <p>Consult the Javadoc for {@link Class#getFields()} for details on
+	 * inheritance and ordering.
```

Comment:
Please remove the statement about ordering since that does not apply to how fields are retrieved (and sorted) in `ReflectionUtils`.

Reply:
Removed

Reply:
In any case, don't worry about it. I'll polish things up after merging into `master` anyway. 😉